tests/docker/Dockerfile

```diff
@@ -32,9 +32,11 @@ ARG ducker_creator=default
 LABEL ducker.creator=$ducker_creator
 
 # Update Linux and install necessary utilities.
-RUN apt update && apt install -y sudo netcat iptables rsync unzip wget curl jq coreutils openssh-server net-tools vim python-pip python-dev libffi-dev libssl-dev cmake pkg-config libfuse-dev iperf traceroute && apt-get -y clean
-RUN python -m pip install -U pip==9.0.3;
-RUN pip install --upgrade cffi virtualenv pyasn1 boto3 pycrypto pywinrm ipaddress enum34 && pip install --upgrade ducktape==0.7.9
+RUN apt-mark hold python2 python2-minimal python2.7 python2.7-minimal libpython2-stdlib libpython2.7-minimal libpython2.7-stdlib
+RUN apt update && apt install -y sudo netcat iptables rsync unzip wget curl jq coreutils openssh-server net-tools vim python3-pip python3-dev libffi-dev libssl-dev cmake pkg-config libfuse-dev iperf traceroute mc && apt-get -y clean
+RUN python3 -m pip install -U pip==20.2.2;
+RUN pip3 install --upgrade cffi virtualenv pyasn1 boto3 pycrypto pywinrm ipaddress enum34
+RUN pip3 install git+https://github.com/confluentinc/ducktape
```

Comment:
Should ducktape no longer be version pinned? (it probably needs to be to avoid future build breakages of old kafka branches).
Or is this just during Python3-ification of ducktape itself?

Reply:
> Should ducktape no longer be version pinned?

No. We should continue to use a specific version of the ducktape.

Currently, master branch of the ducktape contains unreleased fixes for python3.
Please, see the issue for details - https://github.com/confluentinc/ducktape/issues/245

Once fixes will be released I will pin PR to a specific version.

Reply:
Agree. Done.

Reply:
A new ducktape version needs to be released, and this line changed to a versioned pypi install, prior to merge, right?

Reply:
Hello, @edenhill 

Yes, but I'm a bit confused about ducktape release.
Can't receive much feedback from maintainers.

https://github.com/confluentinc/ducktape/issues/245

Reply:
I'll see what I can do.

Reply:
Hello! Any news on ducktape release?